### ISSUE TITLE
need nghttp2 cpp compile for --enable-shared

### ISSUE THREAD

**Issue by belveder79** [leenjewel/openssl_for_ios_and_android#62]
Hi,

as your android arm64 curl build says "need nghttp2 cpp compile" for building shared curl lib, can you elaborate a little on this? How do you build nghttp2 in order to enable a shared curl lib for android arm64?

Thx

**Comment by haoguo** [comment]
Yep, the same problem. When I remove the --disable-shared and build. It says libnghttp2.so not found. Using the -rpath-link not working too. Not sure what happens when building.
Use one without http2 for working around.
With
curl -> 7.76.1
openssl -> 1.1.1k
nghttp2 -> 1.43.0
ndk -> 21.4.7075529

**Comment by sergeymild** [comment]
the same problem

**Comment by belveder79** [comment]
At some point I had this fixed in my fork, replacing the standard build script with very verbose build instructions for cmake...
https://github.com/belveder79/openssl_for_ios_and_android/tree/arth_fix212705
not sure if it is still working...

**Comment by jiek233** [comment]
这是来自QQ邮箱的自动回复邮件。
 
您发送的邮件我已收到，我会在看到后尽快回复您。
